samples/drivers/led/pwm/boards/frdm_mcxc242.overlay

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
```

Comment:
Why do we need to disable the `leds` node, or enable the `pwmleds` one? It looks like this sample would just use the `pwmleds` node anyways, which is already enabled at the board level

Reply:
With leds node enabled, leds do not shine. It is probably because pin function is set to GPIO instead of TPM during initialization. It seems that it does not make sense to have both leds and pwmleds nodes enabled at the same time when using the same leds, because pin configuration depends on initialization order. I propose to disable pwmleds in board dts with a comment and keep the overlay as is. Do you have any comments or proposals, please?

Reply:
Ok, so they are mutally exclusive. Yeah, lets keep the pwmleds off in the board then

Reply:
Changed pwmleds state to disabled in board dts. Updated commit messages.